portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,10 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Dhruvi!</h1>");
-  }
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h1>Hello Dhruvi!!</h1>");
```

Comment:
nit: Is this added by accident?

Reply:
This is what the walkthrough told us to add